app/scripts/controllers/metamask-notifications/utils/utils.ts

```diff
@@ -0,0 +1,621 @@
+import log from 'loglevel';
```

Comment:
This utils is huge. Mostly because was a class.

I've refactored this to instead use modules (makes it much more easier to test!)

FUTURE - lets separate and group utils into different files. E.g. User Storage utils (traversing, mapping, etc); Fetching logic; User Storage mutations.

Reply:
:+1: